app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
The `people` dictionary is a class variable, meaning it is shared among all instances of `Person`. This can lead to issues if multiple `Person` objects with the same name are created, as they will overwrite each other in the dictionary.